web/pdf_page_view.js

```diff
@@ -1006,20 +980,27 @@ class PDFPageView {
 
     // Keep the canvas hidden until the first draw callback, or until drawing
     // is complete when `!this.renderingQueue`, to prevent black flickering.
```

Comment:
Unfortunately it seems that the new implementation doesn't prevent the "black flickering" that this comment talks about since scrolling *very quickly* through the viewer, using the default `tracemonkey.pdf`, will now trigger this effect intermittently.